app/workers/daily_grouping_updater.rb

```diff
@@ -0,0 +1,15 @@
+class DailyGroupingUpdater
+  include Sidekiq::Worker
+
+  def initialize
+    @system_account = Watcher.retrieve_system_account
+    PaperTrail.whodunnit = @system_account
```

Comment:
Would it be preferrable to wrap the whole content of ther `perform` method with a `PaperTrail.whodunnit { } ` block? In the current setup, at the end of this method, the `whodunnit` user has changed.